src/SIOWriter.cc

```diff
@@ -1,7 +1,7 @@
 #include "podio/SIOWriter.h"
+
```

Comment:
```suggestion
```

Reply:
Argh I think some of these are coming from `clang-format`